leshan-core/src/test/java/org/eclipse/leshan/senml/SenMLCborSerializerTest.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *     Boya Zhang - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.leshan.senml;
+
+import org.eclipse.leshan.util.Hex;
+import org.junit.Assert;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SenMLCborSerializerTest extends AbstractSenMLTest {
```

Comment:
Generally, in this project we use "delegation" instead of "inheritance" for reuse code in tests.
But that's not a big deal, if let it like this :)

Reply:
I will take care about it.